scripts/tfsenc_encoding.py

```diff
@@ -0,0 +1,243 @@
+import csv
+import os
+import numpy as np
+import pandas as pd
+import torch
+from numba import jit, prange
+from scipy import stats
+from sklearn.preprocessing import StandardScaler
```

Comment:
nit: imports not in alphabetical order (i can submit a PR to fix this later if you prefer)

Reply:
even better if you could write a precommit hook so this can be streamlined across all users...

Reply:
@VeritasJoker mind if I just add a commit to this pr to fix the import ordering?

Reply:
Yeah go for it. I will add a pr to fix the rest